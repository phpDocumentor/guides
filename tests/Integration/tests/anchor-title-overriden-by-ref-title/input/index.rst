.. _rst-overview:

Overview
**********

RST Overview content


.. _overview-2:
.. _sphinx-overview:

Overview
*********

Sphinx Overview content

Somewhere else
=============

This is a link to the RST Overview: :ref:`RST Overview <rst-overview>`

This is a link to the Sphinx Overview: :ref:`Sphinx Overview <sphinx-overview>`
