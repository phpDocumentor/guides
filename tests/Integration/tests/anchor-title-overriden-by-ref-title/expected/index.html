<!DOCTYPE html>
<html lang="en">
<head>
    <title>Overview</title>

</head>
<body>
    <div class="section" id="rst-overview">
    <h1>Overview</h1>

            <p>RST Overview content</p>
    </div>

<div class="section" id="sphinx-overview">
    <h1>Overview</h1>

            <p>Sphinx Overview content</p>
            <div class="section" id="somewhere-else">
    <h2>Somewhere else</h2>

            <p>This is a link to the RST Overview: <a href="/index.html#rst-overview">RST Overview</a></p>
            <p>This is a link to the Sphinx Overview: <a href="/index.html#sphinx-overview">Sphinx Overview</a></p>
    </div>

    </div>

</body>
</html>
